uobtheatre/productions/models.py

```diff
@@ -445,6 +442,21 @@ def total_capacity(self, seat_group=None):
         response = self.performance_seat_groups.aggregate(Sum("capacity"))
         return response["capacity__sum"] or 0
 
+    def total_capacity(self, seat_group=None):
+        """Total capacity of the Performance.
+
+        The is the total number of seats (Tickets) which can be booked for this
+        performance. This does not take into account any existing Bookings.
+
+        Returns:
+            int: The capacity of the show
+        """
+        if seat_group is not None:
+            return self.total_seat_group_capacity(seat_group=seat_group)
+        if self.capacity:
+            return min(self.capacity, self.total_seat_group_capacity())
```

Comment:
Could refactor this into a one liner
`return min(...) if self.capacity else ...`

Reply:
![image](https://user-images.githubusercontent.com/61627133/132324193-4d27026d-0150-4b5a-8c1d-422bb2b7ec9f.png)

i prefer the left, but will commit the right if you lot prefer

Reply:
I still prefer right but its not a deal breaker